WebDriverAgentLib/FBApplicationProcessProxy.m

```diff
@@ -45,9 +52,16 @@ - (void)waitForQuiescenceIncludingAnimationsIdle:(BOOL)includeAnimations
   [self.applicationProcess waitForQuiescenceIncludingAnimationsIdle:includeAnimations];
 }
 
-- (id)forwardingTargetForSelector:(SEL)aSelector
```

Comment:
Why kill `forwardingTargetForSelector`? it is faster then invocation right?

Reply:
I haven't done any benchmarking, and implemented the only two required methods. if this is faster, then let's use it.

Reply:
From https://developer.apple.com/documentation/objectivec/nsobject/1418855-forwardingtargetforselector
```
This method gives an object a chance to redirect an unknown message sent to it before the much more expensive forwardInvocation: machinery takes over.
```

Reply:
Thanks. Simple benchmark with 10 million invocations answers the question "How ```much more expensive```?" :-) 0.3 seconds vs 26 seconds